tests/conftest.py

```diff
@@ -49,13 +49,15 @@ def temp_plugins_config_file():
 def config(
     temp_plugins_config_file: str, mocker: MockerFixture, workspace: Workspace
 ) -> Config:
-    config = Config()
+    config = Config.build_config_from_env()
     if not os.environ.get("OPENAI_API_KEY"):
         os.environ["OPENAI_API_KEY"] = "sk-dummy"
 
     config.plugins_dir = "tests/unit/data/test_plugins"
     config.plugins_config_file = temp_plugins_config_file
-    config.load_plugins_config()
+    from autogpt.plugins.plugins_config import PluginsConfig
```

Comment:
With local imports here I like to have a comment explaining why (usually to prevent circular imports) so that devs coming along know why. It might be easy to conclude that this is a mistake and it should be at the top.